query/src/main/scala/filodb/query/exec/rangefn/RangeFunction.scala

```diff
@@ -226,14 +227,9 @@ trait ChunkedLongRangeFunction extends TimeRangeFunction[TransientRow] {
 object RangeFunction {
   type RangeFunctionGenerator = () => BaseRangeFunction
 
-
-  def downsampleColsFromRangeFunction(dataset: Dataset, f: Option[RangeFunctionId]): Seq[String] = {
+  def downsampleColsFromRangeFunction(schema: Schema, f: Option[RangeFunctionId]): Seq[String] = {
     f match {
       case None                   => Seq("avg")
-      case Some(Rate)             => Seq(dataset.schema.data.valueColName)
```

Comment:
Don't we have to support these functions now?

Reply:
@TanviBhavsar  if you look a few lines down, I simply moved a whole bunch of them into the `case other =>` which does the same thing.